e2e/tests-dfx/create.bash

```diff
@@ -83,33 +102,33 @@ teardown() {
     assert_match "dns error: failed to lookup address information"
 }
 
-
 @test "create accepts --controller <controller> named parameter, with controller by identity name" {
     dfx_start
     dfx identity new alice
     ALICE_PRINCIPAL=$(dfx --identity alice identity get-principal)
+    ALICE_WALLET=$(dfx --identity alice identity get-wallet)
 
-    assert_command dfx canister create --all --controller alice
+    assert_command dfx canister create --all --controller "${ALICE_PRINCIPAL}"
```

Comment:
Since this test verifies that `create accepts --controller <controller> named parameter, with controller by identity name`, let's leave this line as is (`--controller alice`), and instead add a new test that passes in the principal by ID.

That way this test will still be testing what it says it does (that `--controller` works with an identity name).